Makefile

```diff
@@ -40,6 +40,10 @@ release: $(DEV_STAMP)
 build-translations:
 	$(VENV)/bin/pybabel compile -d ihatemoney/translations
 
+database-revision:
```

Comment:
Should we name this "create-database-revision" ?

Reply:
> Should we name this "create-database-revision" ?

+1, othewise LGTM